packages/fluentui/CHANGELOG.md

```diff
@@ -37,6 +37,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Documentation
 - Use https for Embed and Video examples @Hirse ([#22738](https://github.com/microsoft/fluentui/pull/22738))
 
+### Features
```

Comment:
@GeoffCoxMSFT is this the right spot to add to Changelog

Reply:
yes, but moved the features higher in the changelog.

Reply:
Thank you!
Is this good to be merged then? I did test locally.